openrtb_ext/imp_madvertise.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+// ExtImpMadvertise defines the contract for bidrequest.imp[i].ext.madvertise
+type ExtImpMadvertise struct {
+	PlacementId string `json:"placementId"`
```

Comment:
Nitpick: Go syntax prefers `PlacementID` for the variable name.

Reply:
Done. By the way, about macro logic I replacement Placement by Zone.
[master d4fd6c47] ExtImpMadvertise use ZoneID variable name instead of zoneId